File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OcsaOCS0304U00GrdOCS0305ListInfo.java
package nta.med.data.model.ihis.ocsa;

public class OcsaOCS0304U00GrdOCS0305ListInfo {
	private String memb;
	private String yaksokDirectCode;
	private Double pkSeq;
	private String directGubun;
	private String nurGrName;
	private String directCode;
	private String nurMdName;
	private String directCont1;
	private String directCont2;
	private String directText;
	private String directContYn;

	public OcsaOCS0304U00GrdOCS0305ListInfo(String memb,
			String yaksokDirectCode, Double pkSeq, String directGubun,
			String nurGrName, String directCode, String nurMdName,
			String directCont1, String directCont2, String directText,
			String directContYn) {
		super();
		this.memb = memb;
		this.yaksokDirectCode = yaksokDirectCode;
		this.pkSeq = pkSeq;
		this.directGubun = directGubun;
		this.nurGrName = nurGrName;
		this.directCode = directCode;
		this.nurMdName = nurMdName;
		this.directCont1 = directCont1;
		this.directCont2 = directCont2;
		this.directText = directText;
		this.directContYn = directContYn;
	}

	public String getMemb() {
		return memb;
	}

	public void setMemb(String memb) {
		this.memb = memb;
	}

	public String getYaksokDirectCode() {
		return yaksokDirectCode;
	}

	public void setYaksokDirectCode(String yaksokDirectCode) {
		this.yaksokDirectCode = yaksokDirectCode;
	}

	public Double getPkSeq() {
		return pkSeq;
	}

	public void setPkSeq(Double pkSeq) {
		this.pkSeq = pkSeq;
	}

	public String getDirectGubun() {
		return directGubun;
	}

	public void setDirectGubun(String directGubun) {
		this.directGubun = directGubun;
	}

	public String getNurGrName() {
		return nurGrName;
	}

	public void setNurGrName(String nurGrName) {
		this.nurGrName = nurGrName;
	}

	public String getDirectCode() {
		return directCode;
	}

	public void setDirectCode(String directCode) {
		this.directCode = directCode;
	}

	public String getNurMdName() {
		return nurMdName;
	}

	public void setNurMdName(String nurMdName) {
		this.nurMdName = nurMdName;
	}

	public String getDirectCont1() {
		return directCont1;
	}

	public void setDirectCont1(String directCont1) {
		this.directCont1 = directCont1;
	}

	public String getDirectCont2() {
		return directCont2;
	}

	public void setDirectCont2(String directCont2) {
		this.directCont2 = directCont2;
	}

	public String getDirectText() {
		return directText;
	}

	public void setDirectText(String directText) {
		this.directText = directText;
	}

	public String getDirectContYn() {
		return directContYn;
	}

	public void setDirectContYn(String directContYn) {
		this.directContYn = directContYn;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OcsaOCS0304U00GrdOCS0305ListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OcsaOCS0304U00GrdOCS0305ListInfo
	{
		private String _memb;
		private String _yaksokDirectCode;
		private String _pkSeq;
		private String _directGubun;
		private String _nurGrName;
		private String _directCode;
		private String _nurMdName;
		private String _directCont1;
		private String _directCont2;
		private String _directText;
		private String _directContYn;
		private String _rowState;

		public String Memb
		{
			get { return this._memb; }
			set { this._memb = value; }
		}

		public String YaksokDirectCode
		{
			get { return this._yaksokDirectCode; }
			set { this._yaksokDirectCode = value; }
		}

		public String PkSeq
		{
			get { return this._pkSeq; }
			set { this._pkSeq = value; }
		}

		public String DirectGubun
		{
			get { return this._directGubun; }
			set { this._directGubun = value; }
		}

		public String NurGrName
		{
			get { return this._nurGrName; }
			set { this._nurGrName = value; }
		}

		public String DirectCode
		{
			get { return this._directCode; }
			set { this._directCode = value; }
		}

		public String NurMdName
		{
			get { return this._nurMdName; }
			set { this._nurMdName = value; }
		}

		public String DirectCont1
		{
			get { return this._directCont1; }
			set { this._directCont1 = value; }
		}

		public String DirectCont2
		{
			get { return this._directCont2; }
			set { this._directCont2 = value; }
		}

		public String DirectText
		{
			get { return this._directText; }
			set { this._directText = value; }
		}

		public String DirectContYn
		{
			get { return this._directContYn; }
			set { this._directContYn = value; }
		}

		public String RowState
		{
			get { return this._rowState; }
			set { this._rowState = value; }
		}

		public OcsaOCS0304U00GrdOCS0305ListInfo() { }

		public OcsaOCS0304U00GrdOCS0305ListInfo(String memb, String yaksokDirectCode, String pkSeq, String directGubun, String nurGrName, String directCode, String nurMdName, String directCont1, String directCont2, String directText, String directContYn, String rowState)
		{
			this._memb = memb;
			this._yaksokDirectCode = yaksokDirectCode;
			this._pkSeq = pkSeq;
			this._directGubun = directGubun;
			this._nurGrName = nurGrName;
			this._directCode = directCode;
			this._nurMdName = nurMdName;
			this._directCont1 = directCont1;
			this._directCont2 = directCont2;
			this._directText = directText;
			this._directContYn = directContYn;
			this._rowState = rowState;
		}

	}
}